Makefile

```
@@ -179,22 +173,22 @@ fmt: install-uv ## check the pre-commit hooks and the linting
 	@${UVX_BIN} pre-commit run --all-files
 
 ##@ Releasing and Versioning
-bump: install-uv ## bump version
-	@UV_BIN="${UV_BIN}" /bin/sh "${SCRIPTS_FOLDER}/bump.sh"
+bump: ## bump version
```

checkmake (0.2.2)

[warning] Target body for "bump" exceeds allowed length of 5 (8).

(maxbodylength)